README.md

```diff
@@ -10,6 +10,19 @@ qtc
 go build -o bin/ ./cmd/...
 ```
 
+### Docker
+```shell script
```

Comment:
You'll want `shell-script` or `shell` here, otherwise github won't pick it up: https://github.com/github/linguist/blob/c51a594581b77132866b15d90fbd442a7d0d43d9/lib/linguist/languages.yml#L4943-L4950

Reply:
arg, GoLand refuses to believe either of those exist :(